app/models/supplejack_api/concerns/record.rb

```diff
@@ -31,6 +31,8 @@ module SupplejackApi::Concerns::Record
     scope :suppressed,      -> { where(status: 'suppressed') }
     scope :solr_rejected,   -> { where(status: 'solr_rejected') }
 
+    scope :ready_for_processing, -> { where(processed: false) }
```

Comment:
`processed` is a pretty vague field name. I would have preferred something like `indexed`

Reply:
I wanted a term that covered removals from the index as well.

Reply:
Oh yeah fair enough, tricky.. `index_updated` ?.

Reply:
Yeah that sounds good 👍